app/views/static/home.html.haml

```diff
@@ -1,15 +1,5 @@
 %h1 Welcome to Team Manager!
```

Comment:
Maybe we should center this too for consistency

Reply:
We played ourselves